template/page-events.html.jinja

```diff
@@ -438,7 +438,7 @@
     { "name": "XRPL Grants Info Session: Financial Inclusion Focused",      
     "description": "Join us for a live information session and Q&A on applying to XRPL Grants Wave 7. This session will provide a general overview of the XRPL Grants application for Wave 7, with a focus on Financial Inclusion projects.",
     "type": "info-session",
-    "link": "https://ripple.zoom.us/webinar/register/WN_YdxeVY9_RJG5SVo-LSip6w#/registration",
```

Comment:
Since Wave 7 applications are over does this make sense to still include on the site?

Reply:
We would need an alternative link, information taking from [link ](https://docs.google.com/spreadsheets/d/1CEewx69xeLwNOH_uOv8GqrgV6eSwRDP_CeaCVcLvMys/edit?usp=sharing)